backend/vendor/assets/stylesheets/jquery.jstree/themes/spree/style.scss

```diff
@@ -0,0 +1,216 @@
+/*
+ * jsTree Spree theme 1.0
+ */
+
+.jstree-spree {
+
+  li, li ul {
+    border-color: #ededed;
```

Comment:
Color literals like `#ededed` should only be used in variable declarations; they should be referred to via variable everywhere else.